engines/tinsel/cursor.cpp

```diff
@@ -321,15 +321,18 @@ void Cursor::SetAuxCursor(SCNHANDLE hFilm) {
 
 	DelAuxCursor();		// Get rid of previous
 
-	// WORKAROUND: There's no palette when loading a DW1 savegame with a held item, so exit if so
-	if (!_vm->_bg->BgPal())
-		return;
+	if (TinselVersion < 3) {
+		// WORKAROUND: There's no palette when loading a DW1 savegame with a held item, so exit if so
+		if (!_vm->_bg->BgPal())
+			return;
+
+		assert(_vm->_bg->BgPal()); // no background palette
```

Comment:
I guess a short comment above this about Noir not having palettes helps the readability slightly (given the context of the comment just above).

Reply:
Good idea, added one.